pkg/azurefile/controllerserver.go

```diff
@@ -141,6 +144,14 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 	if retAccount == "" || retAccountKey == "" {
 		return nil, fmt.Errorf("create file share(%s) on account(%s) type(%s) rg(%s) location(%s) size(%d) timeout(3m)", validFileShareName, account, sku, resourceGroup, location, fileShareSize)
 	}
+	if storeAccountKey == "true" {
```

Comment:
this process should be after file share creation successfully

Reply:
`if storeAccountKey != "false" ` is better, then you don't need to set initial value, and by default, should be storeAccountKey

Reply:
done